src/environment.js

```diff
@@ -139,6 +139,10 @@ class Environment {
     return null;
   }
 
+  get targetCommitSha() {
+    return this._env.PERCY_TARGET_COMMIT || null;
```

Comment:
Note: this is consistently inconsistent — we name the other sha env var `PERCY_COMMIT`, and then translate it to "commitSha" internally.